mypy/nodes.py

```diff
@@ -314,8 +314,8 @@ class MypyFile(SymbolNode):
     # If the value is empty, ignore all errors; otherwise, the list contains all
     # error codes to ignore.
     ignored_lines: dict[int, list[str]]
-    # Lines that are statically unreachable (e.g. due to platform/version check).
-    unreachable_lines: set[int]
+    # Lines that were skipped during semantic analysis (would not be type-checked).
```

Comment:
Please keep the examples of a reason to skip: `e.g. due to platform/version check`

Reply:
Also, always-true: 5b6dc5f5e48146e5b1674534065fe4a672a9060d